deploy/operator.antrea.vmware.com_v1_antreainstall_cr.yaml

```diff
@@ -98,26 +158,34 @@ spec:
             {
                 "type": "portmap",
                 "capabilities": {"portMappings": true}
+            },
+            {
+                "type": "bandwidth",
```

Comment:
question: is the cni-install init container already taking care of installing this CNI?

Reply:
I checked script antrea/build/images/scripts/install_cni_kind, it installs bandwidth the same with other plugins

Reply:
Thanks, I just wanted to make sure we were not taking for granted it was already present on the host.